src/components/OrganizationAutocomplete/OrganizationAutocomplete.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { useDebounce } from 'react-use';
+import { Except } from 'type-fest';
+import { AutocompleteField, AutocompleteFieldProps } from '../form';
+import { useOrganizationAutocompleteQuery } from './OrganizationAutocomplete.generated';
+
+type OrganizationAutocompleteProps<
+  T,
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined
+> = Except<
+  AutocompleteFieldProps<T, Multiple, DisableClearable, FreeSolo>,
+  'options' | 'loading'
+>;
+
+export function OrganizationAutocomplete<
+  T,
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined
+>(
+  props: OrganizationAutocompleteProps<T, Multiple, DisableClearable, FreeSolo>
+) {
+  const [value, setValue] = useState('');
+  const [options, setOptions] = useState<any[]>([]);
+  const [selectedOptions, setSelectedOptions] = useState<any[]>([]);
+  const [searching, setSearching] = useState(false);
+
+  // if we don't want to show any initial options, it'd be nice not to have to call the api before input
+  const { refetch } = useOrganizationAutocompleteQuery({
+    variables: {
+      input: {
+        filter: {
+          name: '',
+        },
+      },
+    },
+  });
+
+  useDebounce(
+    async () => {
+      if (value === '') return;
+      const { data } = await refetch({
+        input: {
+          filter: {
+            name: value,
+          },
+        },
+      });
+      setOptions(
+        data?.organizations?.items.map((item) => item.name.value) || []
+      );
+      setSearching(false);
+    },
+    500,
+    [searching, value]
+  );
```

Comment:
It would be better to have this debounce in the AutocompleteField, at least as an option. That way every wrapper doesn't have to implement it.